packages/cursorless-engine/src/cursorlessEngine.ts

```diff
@@ -68,5 +69,7 @@ export function createCursorlessEngine(
     hatTokenMap,
     snippets,
     injectIde,
+    runIntegrationTests: () =>
+      runIntegrationTests(treeSitter, languageDefinitions),
```

Comment:
If we instead expose `treeSitter` and `languageDefinitions`, then we could avoid `runIntegrationTests` ending up in the prod bundle, right?

Reply:
No. The issue is that the test needs to be defined inside cursorless engine; see PR description for more detail. Also notice that treeSitter actually comes from the caller, so exposing it wouldn't actually do anything